src/api.ts

```diff
@@ -906,6 +977,10 @@ function init(dbPath: string): Context {
 
   migrate(db, path.resolve(__dirname, '../prisma/migrations'))
 
+  // Any import with an `active` state on startup most likely failed due to the server process stopping
+  // so we update these import records to have an error state
+  convertActiveImportsToErrorImports(db)
```

Comment:
Just pinning a TODO: when we support importing from an online tile server, we will be able to restart the import on restart. I don't think it makes sense to attempt restarting an mbtiles import - but an import from an online server could end because the user left the app before it completed.